src/blob/index/core.rs

```diff
@@ -370,6 +370,7 @@ pub(crate) trait FileIndexTrait<K>: Sized + Send + Sync {
     ) -> Result<Self>;
     fn file_size(&self) -> u64;
     fn records_count(&self) -> usize;
+    fn blob_size(&self) -> u64;
```

Comment:
`HEADER_VERSION` in this file should be incremented, because index file format has changed